index.html

```diff
@@ -92,7 +92,7 @@ <h3>Project name goes here</h3>
             </ul>
             <div class="viewbutton">
               <p>See this project</p>
-              <i class="fa-solid fa-arrow-right"></i>
+              <i class="fa-solid fa-arrow-right" alt="Arrow-icon"></i>
```

Comment:
- I don't think an alt attribute is used in icons especially font awesome icons. If an icon is added using an inline image element, then it must have an alt attribute.

- Can we kindly remove all `alt` attributes added to your icons that are not images? I mean icons that are not in an `<img>` tag. 🌸